internal/watcher/watcher.go

```diff
@@ -114,18 +104,20 @@ func WatchDirectories(
 // Differential implements WatchFuncProvider. Use NewDifferential to initialize
 // Differential.
 type Differential struct {
-	ignoredExtensions map[string]struct{}
+	allowedExtensions map[string]bool
```

Comment:
It's idiomatic to use a `map[string]struct{}` if the map is only used for lookup
```suggestion
	allowedExtensions map[string]struct{}
```

Reply:
Map usage changed to string slice: https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/ef27e99c2f5e1082e5c5de658675f9d2d2f0e4ee